src/mqt/bench/utils.py

```diff
@@ -119,36 +120,36 @@ def get_examplary_max_cut_qp(n_nodes: int, degree: int = 2):
     return maxcut.to_quadratic_program()
 
 
-class BernoulliA(QuantumCircuit):
+class BernoulliA(QuantumCircuit):  # type: ignore[misc]
```

Comment:
Because otherwise an error is thrown: ` error: Class cannot subclass "QuantumCircuit" (has
type "Any")`

Reply:
I got rid of one of those two defined classes. I think the other one is needed since QuantumCircuit methods are overridden. I moved it to the `ae` benchmark file.